app/controllers/documents_controller.rb

```diff
@@ -3,11 +3,19 @@
 # Document management
 class DocumentsController < ApplicationController
   def index
+    @sorting_params = [params[:sort], params[:direction]]
+    @category = params[:category]
+    join_sql = "INNER JOIN active_storage_attachments ON active_storage_attachments.record_id = documents.id INNER JOIN active_storage_blobs ON active_storage_blobs.id = active_storage_attachments.blob_id"
```

Comment:
Do we really need such a join ? I think we might not, since the `Document` record seems to have all the info we need (`:created_at` and `:filename`)
If we do need it, is there a way to write it without an SQL string, and with options in the `#joins` method ?
And if not, the `join_sql` variable should have a better name. We can see that's SQL, it would be better if the variable name actually reflects 'what' it does, or 'why' it does it